var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -85,6 +85,11 @@ class Openexr(CMakePackage, AutotoolsPackage):
     with when("build_system=cmake"):
         depends_on("cmake@3.12:", type="build")
 
+    @property
+    def libs(self):
+        # Override because libs have different case than Spack package name
+        name = "libOpenEXR*"
+        return find_libraries(name, root=self.prefix, shared=True, recursive=True)
```

Comment:
If we know where libraries are being installed, it would be good to set recursive to `False`